src/controller/java/MatterInteractionClient-JNI.cpp

```diff
@@ -65,3 +65,15 @@ JNI_METHOD(void, invoke)
         ChipLogError(Controller, "JNI IM Invoke Error: %" CHIP_ERROR_FORMAT, err.Format());
     }
 }
+
+JNI_METHOD(void, extenedableInvoke)
```

Comment:
extra E here as well. I think we should grep for `extenedable` and replace with `extendable` everywhere

Reply:
updated, thanks